tests/code_fellows/test_linked_list.py

```diff
@@ -1 +1,23 @@
-# https://codefellows.github.io/sea-python-401d6/assignments/linked_list.html
\ No newline at end of file
+# https://codefellows.github.io/sea-python-401d6/assignments/linked_list.html
+
+from ...code_fellows.linked_list import LinkedList, Node
+
+
+def test_Linked_List():
+    assert LinkedList(["head"]).head.value == "head"
+    assert (
+        LinkedList([3, "three"]).head.value == "head"
+        and LinkedList([3, "three"]).head.next.value == 3
+    )
```

Comment:
This is less than idea in a test.  Always try to assert this simplest thing possible per assertion so that when it fails, it's easy to parse exactly what went wrong.  Especially if the same can be expressed with 2 assertions that are as easy to read and take up basically the same space.

Consider:
```python
linked_list = LinkedList([3, "three"])
assert linked_list.head.value == "head"
assert linked_list.head.next.value == 3
```